homeassistant/components/isy994/services.yaml

```diff
@@ -68,6 +68,76 @@ send_node_command:
             - "fast_off"
             - "fast_on"
             - "query"
+get_zwave_parameter:
+  name: Get Z-Wave Parameter
+  description: >-
+    Request a Z-Wave Device parameter via the ISY. The parameter value will be returned as a entity extra state attribute with the name "ZW_#"
```

Comment:
Using state attributes like this is very unconventional.

Reply:
as an entity extra state attribute

Reply:
Will fix typos when I push the new test for the system health info.

RE: returning parameter as state attribute, I agree it is a bit unconventional, but I didn't see another easy way to return a value from a user request and make it available for use in the state machine/automations; the ISY994 integration doesn't yet have the same per device configuration options that the built-in Z-Wave integrations have--I just wanted to share my reasoning:

1) Similar method is used by the `androidtv` integration [services](https://www.home-assistant.io/integrations/androidtv/#services) to return requested information.
2) PyISY (upstream module) is designed to return new `aux_properties` (state attributes) on the fly as they are sent by the ISY since not all information about a device is available during the initial query. This is an extension of that principle and uses the existing infrastructure the module and integration provides, except that the value is formally requested by the user rather than just sent upon first change.